nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -117,10 +118,11 @@ def __init__(
         subsampling_conv_channels=-1,
         ff_expansion_factor=4,
         self_attention_model='rel_pos',
-        pos_emb_max_len=5000,
         n_heads=4,
+        att_context_size=[-1, -1],
```

Comment:
Dont create lists inside of functions, use None and then check if None and set `att_context_size=[-1, -1]`

Reply:
Done. Thanks!